src/components/Table.svelte

```diff
@@ -35,6 +58,15 @@
     {#if rows}
       {#each rows as row, idx}
         <tr>
+          {#if selectable}
+            <td>
+              <input
+                type="checkbox"
+                on:change={onSelectHandler.bind(undefined, idx)}
```

Comment:
why do we need to bind to `undefined` here?

Reply:
no specific reason
you can use null or {}
its exactly like
`() => onSelectHandler(idx)`